core/impl/src/main/java/com/blazebit/persistence/impl/transform/SizeTransformationVisitor.java

```diff
@@ -359,6 +360,7 @@ private Expression wrapSubqueryConditionally(SubqueryExpression subquery, boolea
             // see https://hibernate.atlassian.net/browse/HHH-9331
             List<Expression> coalesceArguments = new ArrayList<>(1);
             coalesceArguments.add(subquery);
+            coalesceArguments.add(new NullExpression());
```

Comment:
Seems DataNucleus doesn't like this. Please either report the problem to them and add an issue key to `@Category(NoDatanucleus.class)` or fix it somehow. Here is the error: https://travis-ci.org/Blazebit/blaze-persistence/jobs/375353382#L1572